src/AppInstallerSharedLib/AppInstallerSharedLib.vcxproj

```diff
@@ -108,7 +108,9 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
   <ImportGroup Label="ExtensionSettings">
   </ImportGroup>
-  <ImportGroup Label="Shared" />
+  <ImportGroup Label="Shared">
+    <Import Project="..\PureLib\PureLib.vcxitems" Label="Shared" />

```

Comment:
I think the purelib reference was previously added here by mistake. I feel this one was probably not needed